hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/PhoneHomeTest.java

```diff
@@ -612,6 +612,13 @@ public void testReplicatedMapMemoryCost() {
         assertThat(twoMapsMemoryCost).isGreaterThan(oneMapMemoryCost);
     }
 
+    @Test
+    public void testViridianPhoneHomes() {
+        Map<String, String> parameters;
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(null, parameters.get(PhoneHomeMetrics.VIRIDIAN.getRequestParameterName()));
```

Comment:
nit: please switch the parameters of `assertEquals` method

Reply:
@promanenko looking back at this... expected should indeed be null, so actually it's the other code that's doing it wrong; can you elaborate on this?

Reply:
you are right according to Javadoc, but all other tests here have them the other way :) it's a nitpick, just for the uniformity

Reply:
Just use `assertThat` instead

Reply:
thx @frant-hartm , resolving.